src/v2/Apps/Show/__tests__/ShowArtworks.jest.tsx

```diff
@@ -0,0 +1,69 @@
+import { MockBoot } from "v2/DevTools"
+import { Breakpoint } from "@artsy/palette"
+import React from "react"
+import { ShowArtworksRefetchContainer } from "../components/ShowArtworks"
+import { QueryRenderer, graphql } from "react-relay"
+import { ShowArtworks_Query } from "v2/__generated__/ShowArtworks_Query.graphql"
+import { MockPayloadGenerator, createMockEnvironment } from "relay-test-utils"
+import { mount } from "enzyme"
+
+jest.unmock("react-relay")
+jest.mock("v2/Artsy/Router/useRouter", () => ({
+  useRouter: () => ({
+    match: {
+      location: { query: {} },
+    },
+  }),
+}))
+
+describe("ShowArtworks", () => {
+  let env = createMockEnvironment() as ReturnType<typeof createMockEnvironment>
+
+  const getWrapper = (breakpoint = "xs") => {
+    const TestRenderer = () => (
+      <QueryRenderer<ShowArtworks_Query>
+        environment={env}
+        query={graphql`
+          query ShowArtworks_Query($slug: String!) {
+            show(id: $slug) {
+              ...ShowArtworks_show
+            }
+          }
+        `}
+        variables={{ slug: "show-id" }}
+        render={({ props, error }) => {
+          if (props?.show) {
+            return (
+              <MockBoot breakpoint={breakpoint as Breakpoint}>
```

Comment:
One thing to note here is yr only testing mobile layouts, `xs`

Reply:
Yea, for this spec I think it's sufficient to just check that the filter components are rendered. Since we're just reusing the well-tested filter component, I don't know if we need to get more detailed than that.

Also, generally, I almost always just check one breakpoint (usually `lg`), since it's mostly layout changes (which isn't really tested in each specific component that implements a responsive layout). If there was something specific between viewports, then yea I would def agree it's good to test them (even if it's just making sure `LargeCard` and `SmallCard` appear, or whatever).

Reply:
Yup, was mostly commenting thinking that we should check desktop as you say, since that generally contains more features